packages/edit-post/src/components/editor-regions/index.js

```diff
@@ -17,6 +20,57 @@ function EditorRegions( {
 	publish,
 	className,
 } ) {
```

Comment:
This component is meant to be a generic component to show a layout of an editor (sidebar, header, footer, content). It's meant to be used in other packages see #20050 

I think this is probably not the right place for this logic. We may want to move it to a custom hook applied in the `Layout` component instead. (the component using EditorRegions)

Reply:
Thinking better of this, it makes more sense to have it in `VisualEditor` because we don't need to resize notifications or the code editor for that matter.